<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>draw_buck</class>
 <widget class="QWidget" name="draw_buck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QChartView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="onoff_draw">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>353</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始绘图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clean_window">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清除窗口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>V_Mode(voltage)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>V_Mode(peak currrent)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>I_Mode</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="save_data">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>目标电压：V</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="show_temp">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>79</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>温度显示</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
